skaffold.yaml

```diff
@@ -1196,17 +1175,7 @@ profiles:
     value:
     - bash
     - -ec
-    - |
-      echo "Wait until seed is ready"
-      for i in `seq 1 60`;
-      do
-        if kubectl --kubeconfig=$GARDENER_LOCAL_KUBECONFIG get seed $SEED_NAME 2> /dev/null; then
-          break
-        fi
-        echo "Wait until seed gets created by gardenlet"
-        sleep 2
-      done
-      kubectl --kubeconfig=$GARDENER_LOCAL_KUBECONFIG wait --for=condition=gardenletready --for=condition=bootstrapped seed $SEED_NAME --timeout=5m
+    - hack/usage/wait.sh seed "$SEED_NAME" GardenletReady Bootstrapped SeedSystemComponentsHealthy ExtensionsReady BackupBucketsReady
```

Comment:
Same here regarding the `KUBECONFIG` parameter. Additionally, this scenario does not deploy any BackupBucket by default.

Reply:
```suggestion
    - KUBECONFIG=$GARDENER_LOCAL_KUBECONFIG hack/usage/wait.sh seed "$SEED_NAME" GardenletReady Bootstrapped SeedSystemComponentsHealthy ExtensionsReady
```